.eslintrc

```diff
@@ -0,0 +1,30 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+      "skyscanner",
+      "prettier"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "env": {
+    "browser": true,
```

Comment:
Non of the js code in this repo is for the browser. Im pretty sure we dont have any jest tests either.